AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```

Comment:
>You're vandalising English dictionary. Or maybe vandalizing? :>

Ha, our whole project can be considered "vandalism" on the minds of spies.
And not to forget Google themselves, since they still refuse to add a [ciphering indicator](https://code.google.com/p/android/issues/detail?id=5353).